packages/contracts-bedrock/.storage-layout

```diff
@@ -88,21 +88,23 @@
-|----------------------+----------------------------------------+------+--------+-------+------------------------------------------------|
-| finalizedWithdrawals | mapping(bytes32 => bool)               | 51   | 0      | 32    | contracts/L1/OptimismPortal.sol:OptimismPortal |
-+----------------------+----------------------------------------+------+--------+-------+------------------------------------------------+
++----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------+
+| Name                 | Type                                                       | Slot | Offset | Bytes | Contract                                       |
++============================================================================================================================================================+
+| _initialized         | uint8                                                      | 0    | 0      | 1     | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| _initializing        | bool                                                       | 0    | 1      | 1     | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| params               | struct ResourceMetering.ResourceParams                     | 1    | 0      | 32    | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| __gap                | uint256[48]                                                | 2    | 0      | 1536  | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| l2Sender             | address                                                    | 50   | 0      | 20    | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| finalizedWithdrawals | mapping(bytes32 => bool)                                   | 51   | 0      | 32    | contracts/L1/OptimismPortal.sol:OptimismPortal |
+|----------------------+------------------------------------------------------------+------+--------+-------+------------------------------------------------|
+| provenWithdrawals    | mapping(bytes32 => struct OptimismPortal.ProvenWithdrawal) | 52   | 0      | 32    | contracts/L1/OptimismPortal.sol:OptimismPortal |
```

Comment:
Yes- there is only 1 additional storage slot for the `provenWithdrawals` mapping (slot `52`). I placed it after the `finalizedWithdrawals` mapping so that it would not collide with old storage slots.